### PR TITLE
Update translations from Fedora Zanata

### PR DESCRIPTION
**Pull request by cockpituous**
Update translations from Fedora Zanata
 * [x] po-refresh

### CODE REVIEW

**Comment by marusak** [comment]
https://209.132.184.41:8493/logs/pull-13399-20200115-222727-3ef68a64-cockpit-project-cockpit--fedora-30/TestPages-testBasic-fedora-30-127.0.0.2-2801-FAIL.png
This actually may be a problem with translations... Retrying again

**Comment by martinpitt** [comment]
The fedora-30 test seems stuck, but the problem repeats in [fedora-30/firefox](https://209.132.184.41:8493/logs/pull-13399-20200116-062929-3ef68a64-cockpit-project-cockpit--fedora-30-firefox/log.html#162-2). So there's an actual bug there.

File: po/fr.po [hunk 1]
```diff
@@ -58,9 +61,9 @@ msgctxt "storage-id-desc"
 msgid "$0 File System"
 msgstr "$0 Système de fichiers"
 
-#: pkg/systemd/overview-cards/usageCard.jsx:84
+#: pkg/systemd/overview-cards/usageCard.jsx:138
 msgid "$0 GiB / $1 GiB"
-msgstr ""
+msgstr "$0 GiB / $1GiB"
```
Comment (marusak) [inline po/fr.po]: This seems like the issue.

Comment (martinpitt) [reply po/fr.po]: Ironically this is fixed in https://github.com/cockpit-project/cockpit/pull/13395 , but we don't want to land that yet. :grin:

Comment (marusak) [reply po/fr.po]: https://translate.stg.fedoraproject.org/translate/cockpit/master/fr/?q=GiB&checksum=&offset=#history fixed yesterday

File: po/ja.po [hunk 2]
```diff
@@ -7033,33 +7200,50 @@ msgstr "選択したオペレーティングシステムには、必要なメモ
...
+
+#: pkg/systemd/overview-cards/configurationCard.jsx:335
+msgid "The user $0 is not permitted to modify hostnames"
+msgstr ""
+
+#: pkg/systemd/overview-cards/configurationCard.jsx:361
+#, fuzzy
+#| msgid "The user <b>$0</b> is not permitted to modify realms"
+msgid "The user $0 is not permitted to modify realms"
+msgstr "ユーザー <b>$0</b> はレルムを変更することを許可されていません"
```
Comment (marusak) [inline po/ja.po]: This looks wrong as well.

Comment (martinpitt) [reply po/ja.po]: Indeed, someone unfuzzed without applying the difference. We just dropped the `<b>` though, we didn't change the string. I fixed it in weblate.

File: po/ja.po [hunk 3]
```diff
@@ -282,7 +284,7 @@ msgstr "1 分"
 #: pkg/docker/containers-view.jsx:614
 msgid "1 Vulnerability"
 msgid_plural "$0 Vulnerabilities"
-msgstr[0] ""
+msgstr[0] "1 脆弱性"
```
Comment (marusak) [inline po/ja.po]: This also does not seem ideal. (note that jp does not have plurals). Not really sure what would happen if there would be some different number of vulnerabilities. (but this is not why tests are failing).

Comment (marusak) [reply po/ja.po]: https://github.com/cockpit-project/cockpit/pull/13407 fixes this

**Comment by marusak** [comment]
Writing down some notes:
[this line](https://github.com/cockpit-project/cockpit/blob/master/pkg/systemd/overview-cards/usageCard.jsx#L112) is failing. When I replaced it with just something static, it failed in similar fashion on some other place.
`cockpit.ngettext` returns **array** containing one element (`of $0 CPU core` as there is 1 core in VM). But `cockpit.format` expects string, not array. Thats pretty much it. Trying to figure out, why it is array and not string. We are parsing `ja.po` header, for `Plural-Forms` where the only change is semicolon (I'll try to remove it and see).

File: po/ja.po [hunk 4]
```diff
@@ -9,32 +9,35 @@ msgid ""
...
+"Last-Translator: Ludek Janda <ljanda@redhat.com>\n"
+"Language-Team: Japanese <https://translate.stg.fedoraproject.org/projects/"
+"cockpit/master/ja/>\n"
 "Language: ja\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=1; plural=0\n"
-"X-Generator: Zanata 4.6.2\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
```
Comment (marusak) [inline po/ja.po]: So this semicolon is the real issue here. I opened issue about it: https://github.com/mikeedwards/po2json/issues/87
I am not sure though, if it gets fixed/gets fixed in some reasonable amount of time. Maybe we can add some hack into bots, that would strip this semicolon? (it works always without it and we even add it back there in our `po/po2json` script).

File: po/ko.po [hunk 5]
```diff
@@ -1,40 +1,42 @@
...
+"Last-Translator: Ludek Janda <ljanda@redhat.com>\n"
+"Language-Team: Korean <https://translate.stg.fedoraproject.org/projects/"
+"cockpit/master/ko/>\n"
 "Language: ko\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=1; plural=0\n"
-"X-Generator: Zanata 4.6.2\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
```
Comment (marusak) [inline po/ko.po]: The same issue is also in korean.

File: po/ja.po (hunk 4 above)
Comment (marusak) [reply po/ja.po]: Or could it be, that this was known issue for long time and Zanata was not updating this and nor is Weblate, but when @ludekjanda pushed `.po` files they contained this semicolon?

Comment (marusak) [reply po/ja.po]: Ok, this is on Weblate - https://github.com/cockpit-project/cockpit-weblate/commit/fa85b89a11cd9b6d6eeded161252f54e30f02094

**Pull request closed without merging**